01/components/Table.js

```diff
@@ -2,17 +2,36 @@ import React from 'react';
 import TableHeader from './TableHeader';
 import TableBody from './TableBody';
 import TableFooter from './TableFooter';
+import TableRow from './TableRow';
 
 class Table extends React.Component {
-	render() {
+	renderRows() {
+		const { data } = this.props;
+
+		return data.map(({ id, name, price, quantity }) => {
```

Comment:
To nadal nie jest elastyczne - już do `table` należałoby określić jakie kolumny mają być zaprezentowane ;)